packages/extension-ui/src/Popup/Accounts/EditAccountMenu.tsx

```diff
@@ -137,6 +139,13 @@ function EditAccountMenu({
           />
         </LinksList.Group>
         <LinksList.Group>
+          {connectedActiveTabUrl && (
+            <LinksList.Item
+              onClick={goTo(`/url/manage?url=${connectedActiveTabUrl}`)}
+              rightIcon='chevron'
+              title={t<string>('Disconnect account')}
```

Comment:
Change account connection? Connect / Disconnect account?

Reply:
Hmm, I supposed it could be done this way, but:
1. The way I thought about it was that this menu option only appears when you are connected, so currently `Connect` doesn't make sense. I can of course change it to alwasy appear, but...
2. ...is it even possible to trigger connection manually? I thought you always need to have the dapp send a request to connect.

Reply:
There are two things I guess, whether the app is connected and whether the account is connected (I should probably say injected / approved?).

Since the link is in account settings I thought it main purpose is to change account between injected / not injected while dapp is connected.

So I guess the text should either say `Connect / Disconnect account` (I prefer this option, but maybe it's misleading?) or `Disconnect app / website`